src/Http/Controllers/Users/Conversations/ConversationController.php

```diff
@@ -36,4 +37,24 @@ public function index($userId): JsonResponse
             config('gossip.resources.conversation'),
         );
     }
+
+    /**
+     * Leave a conversation
+     * 
+     * @param  int|string  $conversation
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function destroy($conversationId): JsonResponse
+    {
+        $conversation = $this->getModel($conversationId);
+        $user = auth()->user();
+
+        if($user->conversations->contains($conversation)) {
```

Comment:
Maybe this can be put in a policy? That way an admin could also remove users from a conversation.

Reply:
Currently with this code an admin wont be able to remove another user from a conversation because the user that is being removed from the conversation is the authenticated user. So when an admin makes a request, the admin will get removed from the conversation. There is no extra parameter to remove another user. If this is wanted I need to rewrite it a bit.

Reply:
That's a good point. Let's keep it like this for now then. In the feature we might add a `/conversations/{conversation}/leave/{user}` route.